manim/mobject/geometry.py

```diff
@@ -1264,6 +1268,46 @@ def construct(self):
 
                 self.add(Group(g1, g2, g3).arrange(buff=2))
 
+
+    .. manim:: ArrowExample
+        :save_last_frame:
+
+        class ArrowExample(Scene):
+            def construct(self):
+                left_group = VGroup()
+                # As buff increases, the size of the arrow decreases.
+                for i in np.arange(0, 2.2, 0.45):
+                    left_group += Arrow(buff=i, start=2 * LEFT, end=2 * RIGHT)
```

Comment:
A slightly more descriptive var name.
```suggestion
                for buff in np.arange(0, 2.2, 0.45):
                    left_group += Arrow(buff=buff, start=2 * LEFT, end=2 * RIGHT)
```